tests/functional/tests.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from subprocess import call
-from raven.utils.compat import TestCase
+from raven.utils.testutils import BaseTestCase as TestCase
```

Comment:
Yeah that's true, I assumed since this is a test file all the test dependencies will be available. I can change that if needed.